opennms-webapp/src/main/webapp/admin/notification/noticeWizard/choosePath.jsp

```diff
@@ -165,7 +200,37 @@
             </div>
           </td>
         </tr>
-        <tr>
+
+          <tr>
+              <td width="10%" valign="top" align="left">
+                  <label>Parameter:</label>
```

Comment:
@christianpape What are those `parameters` used for? I only use them for Slack/Mattermost. The docs only tell about notification parameters in the Slack/Mattermost section. But that doesn't mean anything ;-)
Is there an other usage possible? If it's only for Slack/Mattermost I would suggest to rename it accordingly. If it's a generic thing, we should keep the current name.

Reply:
It's a generic thing. So, parameter seems to be fine for me.

Reply:
Ok. Thanks!